src/lib/containers/table/datasets/DatasetItemsEditor.tsx

```diff
@@ -143,22 +147,55 @@ export function DatasetItemsEditor(props: DatasetItemsEditorProps) {
     }
   })
 
-  function addItemsToDataset(newItems: Reference[]) {
+  function addItemsToDataset(itemsToAdd: Reference[]) {
```

Comment:
Just renaming the parameter, `itemsToAdd` is a little more accurate. The items are not necessarily new.